arch/arm/mach-imx/Makefile

```diff
@@ -56,6 +56,9 @@ obj-$(CONFIG_IMX_BOOTAUX) += imx_bootaux.o
 endif
 obj-$(CONFIG_SATA) += sata.o
 obj-$(CONFIG_IMX_HAB)    += hab.o
+ifeq ($(CONFIG_ARCH_MX6),y)
```

Comment:
As this will be needed with other families, should we configure based on the soc, like:

`ifeq ($(SOC),$(filter $(SOC),mx6 mx7ulp))`

Easier to extend later on.

Reply:
Let me take a quick look at combining this check with the imx7ulp

Reply:
Adjusted the Makefile check for mx6.  This should make it much easier in the future to extend support by SoC.